core/Array.carp

```diff
@@ -220,14 +220,20 @@ If the element is not found, returns `Nothing`")
           (set! c (Int.inc c))))
       c))
 
+  (doc unsafe-nth-value "returns the value at index `i` of an array `a` (just like unsafe-nth) but does not take its reference, and does *not* copy the value. Should only be used for optimizations and when you know what you're doing, circumvents the borrow checker!")
```

Comment:
Could we make the reference to `unsafe-nth` a link here (using `[unsafe-nth](#unsafe-nth)`)?